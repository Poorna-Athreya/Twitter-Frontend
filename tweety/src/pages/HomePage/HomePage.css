.sync-button{
    border-style: none;
    background-color: var(--accent-color);
    color:white;
    border-radius: 25px;
    padding: 5px;
    width: 150px;
    height:50px;
    font-size: 30px;
    font-family: 'Poppins',sans-serif;
    font-weight: 700;
}
.homepage-main-para{
    font-size: large;
    font-weight: 200;
}
.sync-button:hover{
    background-color: white;
    color:#4c63AA
}
.root-container{
    display:flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    height: 720px;
    font-size: 30px;
    font-weight: 200;
}