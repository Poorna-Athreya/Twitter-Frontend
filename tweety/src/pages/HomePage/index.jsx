import React from 'react';
import { useNavigate } from 'react-router-dom';
import './HomePage.css';

function HomePage() {
  const navigate = useNavigate();
  return (
    <div className="root-container">
      <p className="hompage-main-para"> Welcome to Tweety</p>

      <button
        className="sync-button"
        type="submit"
        onClick={() => {
          navigate('/records');
        }}

      >
        users
      </button>

    </div>

  );
}
export default HomePage;
