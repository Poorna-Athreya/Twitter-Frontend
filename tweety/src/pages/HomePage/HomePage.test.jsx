import React from 'react';
import { render } from '@testing-library/react';
import HomePage from './index';

const mockNavigate = jest.fn();
jest.mock('react-router-dom', () => ({ useNavigate: () => mockNavigate }));

describe('HomePage', () => {
  it('should take a snapshot of HomePage', () => {
    const { asFragment } = render(<HomePage />);

    expect(asFragment(<HomePage />)).toMatchSnapshot();
  });
});
