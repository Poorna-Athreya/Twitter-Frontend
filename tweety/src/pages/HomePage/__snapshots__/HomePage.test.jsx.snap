// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should take a snapshot of HomePage 1`] = `
<DocumentFragment>
  <div
    class="root-container"
  >
    <p
      class="hompage-main-para"
    >
       Welcome to Tweety
    </p>
    <button
      class="sync-button"
      type="submit"
    >
      users
    </button>
  </div>
</DocumentFragment>
`;
